cloudevents/emitter.go

```diff
@@ -0,0 +1,60 @@
+package cloudevents
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudevents/sdk-go/v2/client"
+	"github.com/cloudevents/sdk-go/v2/event"
+	"github.com/cloudevents/sdk-go/v2/protocol/http"
+	"github.com/cloudevents/sdk-go/v2/types"
+	"github.com/google/uuid"
+)
+
+type Emitter struct {
+	Endpoint    string
+	Source      string
+	Type        string
+	Id          string
+	Data        string
+	ContentType string
+}
+
+func NewEmitter() *Emitter {
+	return &Emitter{
+		Source:      "/boson/fn",
+		Type:        "boson.fn",
+		Id:          uuid.NewString(),
+		Data:        "",
+		ContentType: event.TextPlain,
+	}
+}
+
+func (emitter *Emitter) Emit(endpoint string) (err error) {
```

Comment:
Make `ctx` the first param of the function.

Reply:
Go idiom is to have the funciton receiver be a single letter.  I used to find this draconian but have come to appreciate the consistency.

```suggestion
func (e *Emitter) Emit(endpoint string) (err error) {
```